README.md

```diff
@@ -11,30 +11,32 @@ You can try out Recursive Extractor [in your browser](https://microsoft.github.i
 # Supported File Types
 | | | |
 |-|-|-|
-| 7zip | ar | bzip2 |
-| deb | gzip | iso |
-| rar | tar | vhd |
-| vhdx | vmdk | wim |
-| xzip | zip |  |
+| 7zip | ar   | bzip2 |
+| deb  | gzip | iso   |
+| rar  | tar  | vhd   |
+| vhdx | vmdk | wim   |
+| xzip | zip  |       |
 
 # Usage
-This example will print out the paths of all the files in the archive.
+This this code adapted from the Cli extracts the contents of given archive located at `options.Input`
+to a directory located at `options.Output` and prints the relative path of each file inside the archive.
 ```csharp
-var path = "/Path/To/Your/Archive"
 var extractor = new Extractor();
-try {
-    IEnumerable<FileEntry> results = extractor.ExtractFile(path);
-    foreach(var found in results)
-    {
-        Console.WriteLine(found.FullPath);
-    }
-}
-catch(OverflowException)
+var extractorOptions = new ExtractorOptions()
 {
-    // This means Recursive Extractor has detected a Quine or Zip Bomb
+    ExtractSelfOnFail = true,
+    Parallel = true,
+};
+foreach(var result in extractor.ExtractFile(options.Input, extractorOptions))
+{
+    Directory.CreateDirectory(Path.Combine(options.Output,Path.GetDirectoryName(result.FullPath)?
+        .TrimStart(Path.DirectorySeparatorChar) ?? string.Empty));
+    using var fs = new FileStream(Path.Combine(options.Output,result.FullPath), FileMode.Create);
```

Comment:
See comment [here](https://github.com/microsoft/RecursiveExtractor/pull/22/files#r471034502).

Reply:
This is safe because FullPath is ZipSlip safe based on latest commit.  See how FullPath is set in FileEntry.